README.md

```diff
@@ -158,90 +166,27 @@ squaredNumbers;
-//    ]
-
-newEngineers;
-// => [
-//      { userID: 15, title: "Developer Apprentice", accessLevel: "admin" },
-//      { userID: 63, title: "Developer Apprentice", accessLevel: "admin" },
-//      { userID: 97, title: "Developer Apprentice", accessLevel: "admin" },
-//      { userID: 12, title: "Developer Apprentice", accessLevel: "admin" },
-//      { userID: 44, title: "Developer Apprentice", accessLevel: "admin" }
-//    ]
-```
-
-As before, we are calling our version of the `map()` function and passing in the
-collection and a callback. Notice that we're using `Object.assign()` to create a
-**new** object with updated values instead of mutating the original object's
-`accessLevel` property. Nondestructive updating is an important concept to
-practice — destructively modifying objects at multiple points within a code base
-is one of the biggest sources of bugs.
-
-Next, we just need a simple array of the new engineers' `userID`s that we can
-shoot over to the system administrator:
+const originalNumbers = [1, 2, 3, 4, 5];
 
-```js
-const userIDs = map(newEngineers, function (eng) {
-  return eng.userID;
+const squaredNumbers = originalNumbers.map(function (num) {
+  return num * num;
 });
 
-userIDs;
-// => [15, 63, 97, 12, 44]
-```
-
-Finally, we'll update our engineer objects to indicate that all the new
-engineers have been provided a new work laptop. This time, though, let's use
-JavaScript's built-in `Array.prototype.map()` method:
-
-```js
-const equippedEngineers = newEngineers.map(function (eng) {
-  return Object.assign({}, eng, { equipment: "Laptop" });
-});
+originalNumbers;
+// => [1, 2, 3, 4, 5]
 
-equippedEngineers;
-// => [
-//      { userID: 15, title: "Developer Apprentice", accessLevel: "admin", equipment: "Laptop" },
-//      { userID: 63, title: "Developer Apprentice", accessLevel: "admin", equipment: "Laptop" },
-//      { userID: 97, title: "Developer Apprentice", accessLevel: "admin", equipment: "Laptop" },
-//      { userID: 12, title: "Developer Apprentice", accessLevel: "admin", equipment: "Laptop" },
-//      { userID: 44, title: "Developer Apprentice", accessLevel: "admin", equipment: "Laptop" }
-//    ]
+squaredNumbers;
+// => [1, 4, 9, 16, 25]
 ```
 
-Note how similar this method call is to the one using our version of `map()`:
-the only difference is that we call the built-in `.map()` method _on_ our array,
-rather than passing the array as an argument. There _is_ one big difference
-between the two, though: we didn't have to do all the work of building
-`Array.prototype.map()`!
-
-Now that we understand how the built-in `.map()` array method is implemented, we
-can stick to the native method and get rid of our copycat `map()` function.
+That's it! This code works exactly the same as our version - the only difference
+is we called the method on our array rather than passing the array as an
+argument. And, of course, we didn't have to code `map()` ourselves!
 
```

Comment:
For consistency, maybe add a brief conclusion? If it seems too repetitive at this point, would still be fine to leave out.